articles/docfx.json

```diff
@@ -64,6 +64,10 @@
         }
       }
     },
+    "titleSuffix": {
+      "*.md": "Microsoft Quantum"
+      } 
```

Comment:
Similarly here:
```suggestion
```

Reply:
Validation complete and fix submitted. Thank you!